src/components-examples/material/tree/tree-nested-overview/tree-nested-overview-example.css

```diff
@@ -8,3 +8,19 @@
   margin-bottom: 0;
   list-style-type: none;
 }
+
+/*
+ * This padding sets alignment of the nested nodes.
+ */
+.example-tree .mat-nested-tree-node div[role=group] {
+  padding-inline-start: 40px;
```

Comment:
`padding-inline-start` doesn't appear to be supported in a lot of browsers. Can it be changed to `padding-left` instead?

Also I believe there's the `matTreeNodePadding` directive which is supposed to deal with the padding on tree nodes. It's used in some of the other examples.

Reply:
Tied matTreeNodePadding but that didn't work with nestedTreeNode. I will check why it doesn't work.

Reply:
matTreeNodePadding works with the FlatTree data source since it needs node level as the input.